formcentral/server.go

```diff
@@ -72,31 +68,14 @@ func (s *Server) CreateTranslator(c echo.Context) error {
 	return c.JSON(http.StatusOK, translator)
 }
 
-type Config struct {
-	Db       string `env:"CHATBASE_DATABASE,required"`
-	User     string `env:"CHATBASE_USER,required"`
-	Password string `env:"CHATBASE_PASSWORD,required"`
-	Host     string `env:"CHATBASE_HOST,required"`
-	Port     string `env:"CHATBASE_PORT,required"`
-}
-
-func getConfig() Config {
-	cfg := Config{}
-	err := env.Parse(&cfg)
-	handle(err)
-	return cfg
-}
-
 func handle(err error) {
 	if err != nil {
 		log.Fatal(err)
 	}
 }
 
 func main() {
-	cfg := getConfig()
-	pool := getPool(&cfg)
-
+	pool := getPool()
```

Comment:
Not a big deal, but I do feel like I have a slight preference for configs getting passed into things rather than them relying on a function that picks things from the environment. In this case, there's no difference, but in general it can make testing easier if we want to manually change the config for testing.